toolset/setup/linux/frameworks/cppcms.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+fw_installed cppcms && return 0
+
+#http://cppcms.com/wikipp/en/page/cppcms_1x_build
+
+#note the -rc1 in url
+VERSION=1.1.1
+BACKNAME=cppcms
+CPPCMS_HOME=$IROOT/$BACKNAME-$VERSION
+
+sudo apt-get update -qq
+sudo apt-get install -y cmake libpcre3-dev zlib1g-dev libgcrypt11-dev libicu-dev python
```

Comment:
Same comment here about hoisting to `fw_depends`, and specify which version of `python` is required (e.g. `fw_depends python3`).